src/styles/components/_jca-code-reference.scss

```diff
@@ -0,0 +1,36 @@
+.jca-code-reference {
+  border-radius: $default_radius;
+  background: $white;
+  overflow: hidden;
+  display: flex;
+  box-shadow: $default-shadow;
+  margin-bottom: 20px;
+
+  &__example {
+    flex: 1;
+    min-width: 50%;
+  }
+
+  &__description {
+    padding: $default_padding;
+    background-color: #083358;
+    color: $white;
+  }
+
+  &__visual {
+    padding: $default_padding;
+  }
+
+  &__code {
+    background-color: $black;
+    flex: 1;
+    min-width: 50%;
+    padding: $default_padding;
+
+    :not(pre) > code[class*="language-"],
+    pre[class*="language-"] {
+      background: none;
+      font-size: 1.1em;
+    }
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5693353)